module/ArmMotion.h

```diff
@@ -1,23 +1,38 @@
 /**
  * @file ArmMotion.cpp
  * @brief アーム動作クラス
- * @author kodama0720, yutotanaka24
+ * @author kodama0720, yutotanaka24, hiroto0927, miyashita6
```

Comment:
変更しなくても処理に影響は出ないとは思いますが他コードと一貫性を持たせた方がいいかもしれません
```suggestion
 * @author kodama0720, yutotanaka24, hiroto0927, miyashita64
```